src/library_wasmfs.js

```diff
@@ -1,3 +1,65 @@
-var WasmfsLibrary = {}
+var WasmfsLibrary = {
+  $FS : {
+    preloadedFiles: [],
+    preloadedDirs: [],
+    // TODO: Clean up the following functions - currently copied from library_fs.js directly.
+    createPreloadedFile: function(parent, name, url, canRead, canWrite, onload, onerror, dontCreateFile, canOwn, preFinish) {
+        Browser.init(); // XXX perhaps this method should move onto Browser?
```

Comment:
I think the `$Browser` stuff is just because this is temporarily copy-pasted from the main FS code. When we clean it up we'd definitely remove that. (No harm in removing it now, though, too.)

For the global `FS` object, that is also mainly as a temporary way to easily get up and running, by emulating the same `FS.*` APIs that the file packager expects. Otherwise we'd need to rewrite the file packager right now. But we will definitely get to that before declaring WASMFS stable.